marketplace/optimizely/package.json

```diff
@@ -40,5 +42,10 @@
     "last 5 Chrome version",
     "> 1%",
     "not ie <= 11"
-  ]
+  ],
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(css|less)$": "identity-obj-proxy"
```

Comment:
@suevalov I found that I still needed this, otherwise CSS would fail. Potentially we need to change this?

https://github.com/contentful/create-contentful-extension/blob/master/packages/contentful-extension-scripts/scripts/utils/createJestConfig.js#L15